tools/rest-api-docs/authentication.md

```
@@ -0,0 +1,24 @@
+---
+title: Authentication
+sidebar_position: 0
+description: How to authenticate with the REST API
+---
+
+# Authentication
+
+The REST API uses **Bearer token** authentication. Include your token in every request:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)